### ISSUE TITLE
Stacksgov GitHub Organization Cleanup

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/Stacks-Grant-Launchpad#30]
*We prefer clear, concrete, and concise applications. If an application is incomplete or unclear, we will request an update to the application.*

## Background

> What problems do you aim to solve? How does it serve the mission of a user owned internet?

Since the Governance Working Group first formed back in February, the repositories used by the group have undergone some significant changes, and both the information and infrastructure need some attention.

The resources repository is the go-to location [for research](https://github.com/stacksgov/resources/tree/master/research), past and present, historical data including [stakeholder interviews](https://github.com/stacksgov/resources/tree/master/interviews) and the [governance survey](https://github.com/stacksgov/resources/tree/master/survey), and a table of [past call information](https://github.com/stacksgov/resources/tree/master/calls) including meeting agendas, videos, notes, and attached resources for both working group and monthly engineering calls.

The project management (pm) repository contains a [number of issues](https://github.com/stacksgov/pm/issues) centered around meetings, resource updates, and notes requests. The [standard operating procedures](https://github.com/stacksgov/pm/blob/master/sop.md) serve as supplement to the [org-wide contributing guide](https://github.com/stacksgov/.github/blob/master/CONTRIBUTING.md).

Both research and pm have their own websites generated [by docsify](https://docsify.js.org/#/) using the content inside each repo and hosted on GitHub pages:

- https://stacksgov.github.io/resources
- https://stacksgov.github.io/pm

There are also [proposals](https://github.com/stacksgov/proposals), [updates](https://github.com/stacksgov/updates), [community](https://github.com/stacksgov/community), and now [Stacks-Grants](https://github.com/stacksgov/Stacks-Grants). 

## Project Overview

> What solution are you providing? Who will it serve?

This grant would be used to review the information across all outlets in full, similar to [the planning and implementation](https://github.com/stacksgov/pm/issues/77#issuecomment-655621179) when things were reorganized back in July, and use that to make a comprehensive update that:

- archives and preserves historical content
- archives and closes stale or unused content
- updates and organizes governance research
- updates past call information, including uploading recorded videos to YouTube
- updates standard operating procedures to make governance more accessible and consistent for anyone involved
- researches and implements updated branding based on [this issue](https://github.com/stacksgov/pm/issues/115) and the [new brand guidelines](https://www.stx.design/) for the ecosystem
- revamps the docsify implementation, [fixing errors](https://github.com/stacksgov/pm/issues/104) and reviewing the addition of new features, such as better navigation

Additional goals can be named along the way and community feedback is definitely encouraged. This information, these repositories, and their related structures help encourage community involvement with governance of the protocol, and facilitate healthy conversations about how to govern the future of a user-owned Internet.

## Scope

> What are the components or technical specs of the project? What will the final deliverable look like? How will you measure success?

Each goal listed in the overview can be further broken down into individual tasks, some of which are outlined in existing issues, and progress will be tracked via GitHub issues and a project board at the organization level.

Any content developed or revised should be as evergreen as possible, with navigation that works both natively through the GitHub readmes, as well as with the generated docsify websites.

Success would be an up-to-date set of repositories with well-documented procedures to continue the process, such that anyone who wants to be involved could take part in the initiative. Tasks should be divided up into small, consumable goals for working group members, ideally completed on a regular schedule, while producing coherent and consistent reference material for the community moving forward.

## Budget and Milestones

> What grant amount are you seeking? How long will the project take in hours? If more than 20, please break down the project into milestones, with a clear output (e.g., low-fi mockup, MVP with two features) and include the estimated work hours for each milestone.

### Total Grant Request: $1,500
- for an estimated 28 hours of work
- to be completed over 6 weeks
- averaging 1 week per milestone

**M1: fixing what's broken - 4hrs $215**
- fix docsify link issues stacksgov/pm#104
- review docsify configuration in full
- document docsify file structure
- document docsify features in use

**M2: updating call content - 8hrs $430**
- identify all missing call data
- obtain any call recordings not saved/uploaded
- upload call recordings to Youtube channel
- update call tables for governance working group
- update call tables for engineering calls
- document procedure for Youtube channel uploads
- document procedure for updating call tables

**M3: reorganizing information - 4hrs $215**
- review, move, and archive past content
- close old or stale issues, open new issues if necessary
- review .github resources for all repos
- document structure and usage of .github repo
- review navigation flow between all content

**M4: rebranding - 4hrs $215**
- develop new logo based on STX design standards
- circulate with community for input
- upon approval: update across all outlets

**M5: setup community accounts - 4hrs $215**
- verify zoom pro account for community use
- review subdomain usage for docsify sites
- review access and ownership of Youtube channel
- document list of shared accounts
- document how to request access
- document how to see who currently has access

**M6: 30 foot view - 4hrs $215**
- create new SOP including all items documented above
- create or update any remaning documentation for working group participation
- implement docsify navigation and TOCs based on new navigation flow

## Team

> Who is building this? What relevant experience do you bring to this project? Are there skills sets you are missing that you are seeking from the community?

Jason Schrader / [@whoabuddy](https://github.com/whoabuddy). I am currently the main contributor and maintainer for the @stacksgov organization, and have made several updates throughout the year as noted in the screenshots below.

![20201118_002_Selection](https://user-images.githubusercontent.com/9038904/99695460-d4bc7880-2a4a-11eb-9c73-6d1a4e635ef6.png)

![20201118_003_Selection](https://user-images.githubusercontent.com/9038904/99695469-d7b76900-2a4a-11eb-9488-e925d01253cf.png)

I helped develop the [labeling system for issues](https://github.com/stacksgov/pm/issues/67), implement the [docsify integration](https://github.com/stacksgov/pm/issues/69) for resources and project management, and recently updated the [standard operating procedures](https://github.com/stacksgov/pm/pull/100).

I have also written several tutorials and educational content, most of which can be [found on Sigle](https://app.sigle.io/whoabuddy.id.blockstack):
- [Blockstack: How to -git- Involved](https://app.sigle.io/whoabuddy.id.blockstack/s0zMy6UW3cTOfR-Lm9dCg)
- [Step 2: Running a Miner Node (Zero to Testnet Series)](https://app.sigle.io/whoabuddy.id.blockstack/Lqq5_aeI1A06l_FQ8s9Jv)
- [Decoding Discord Chats and Hangouts](https://app.sigle.io/whoabuddy.id.blockstack/IFT64oGIRG6yvFF4LB0UW)

## Risks

> What dependencies or obstacles do you anticipate? What contingency plans do you have in place?

This grant will require some collaboration in order to secure the missing content as well as setup shared accounts for community usage.

The Stacks Foundation will need to assist with the creation of a Zoom Pro account for community use ([stacksgov/pm#14](https://github.com/stacksgov/pm/issues/14#issuecomment-714635255)), as well as verify if subdomains for the related websites are a desired feature (stacksgov/pm#79), and if so, grant access or make the configuration changes to enact it.

Blockstack PBC will need to assist with providing any missing Zoom recording links, so the videos, transcripts, and other materials can be downloaded then uploaded to both GitHub and Youtube.

I do not anticipate any obstacles or blockers as part of this process, but in the event something is not accessible or unavailable, it will be noted as part of the process and will not affect the overall completion of the grant.

## Community and Supporting Materials

> Do you have previous projects, code commits, or experiences that are relevant to this application? What community feedback or input have you received? How do you plan to share your plan to the community over time and as the final deliverable?

This grant application will be shared as an action item in the upcoming Stacksgov Working Group meeting (stacksgov/pm#113).

I plan to share updates after each milestone in the #governance Discord channel, as well as a quick update during each governance working group meeting until the grant is completed.

**Comment by RaffiSapire** [comment]
Thank you, your grant request is approved! The next step is to email grants@stacks.org to receive a link to the contract and we'll submit payment for M1. 

Thanks Jason and we're grateful to have you contributing.

**Comment by whoabuddy** [comment]
I am happy to report that the first payment arrived and I plan to start this project on Monday, December 21st, 2020.

There is a project board in the stacksgov/pm repository where I will be tracking progress and filing related issues:

- GitHub Cleanup and Reorganization: https://github.com/stacksgov/pm/projects/3

There are also three labels to help identify issues related to the grant and their individual status:

- JS-in-progress: https://github.com/stacksgov/pm/labels/JS-in-progress
- JS-grant-backlog: https://github.com/stacksgov/pm/labels/JS-grant-backlog
- JS-grant-complete: https://github.com/stacksgov/pm/labels/JS-grant-complete

Updates will be shared during each Governance Working Group meeting.

Please let me know if you have any questions or need any additional information!

**Comment by RaffiSapire** [comment]
Hello! Thank you for all your hard work this year. We would love to send you a holiday note and some custom stacks stickers. Please email me at Raffi@stacks.org with your name/addresses if you'd like to receive one. We really appreciate all your help and contributions. Stacks is what it is because of you!

**Comment by whoabuddy** [comment]
## Update: M1 is complete!

merged 3 PRs :tada: 

- stacksgov/pm#122
- stacksgov/pm#123
- stacksgov/resources#49

Some things fixed along the way:

- changed Blockstack to Stacks
- updated forum links to https://forum.stacks.org
- removed work-stream content from pm (unused)
- fixed issue templates typo (preview below - for when someone clicks on "new issue")

![20210106_003_Selection](https://user-images.githubusercontent.com/9038904/103912390-79128100-50c4-11eb-800f-04a13bb4e1f3.png)

**Comment by RaffiSapire** [comment]
Awesome! Thanks, I sent M1+M2 payments so will await your update to disburse M3.

**Comment by whoabuddy** [comment]
## Update: M2 is complete!

merged 3 PRs :tada: 

- stacksgov/pm#130
- stacksgov/resources#50
- stacksgov/resources#51

one PR open for discussion :thought_balloon: 

- stacksgov/pm#131

Next up will be M3 + M4, which can happen simultaneously and have been partially addressed by the first two.

Things are moving right along! (albeit, a bit slower than expected)

**Comment by RaffiSapire** [comment]
M3+M4 have been disbursed! Thank you

**Comment by whoabuddy** [comment]
I created a survey for feedback on two new logo designs, and included an open call to any designers who think they can do better! :)

I've shared this in the Stacks Discord as well, and will follow up on it in the next meeting until we get a decent number of responses.

https://survey.absorbingchaos.com/survey/189ZWJqLxL1ZhFPMfLKfUoHfDePdLex7cL/332c90c5-454e-4afc-90a4-578ed9ad4a8b

**Comment by RaffiSapire** [comment]
Hi @whoabuddy wanted to check in on the status of this grant?

**Comment by whoabuddy** [comment]
Hi @RaffiSapire, thanks for checking in!

## M3 is complete :tada: 

- https://github.com/stacksgov/pm/pull/137
- https://github.com/stacksgov/resources/pull/53
- https://github.com/stacksgov/pm/issues/136

I closed out a lot of older issues and implemented the stale bot. Everything is down to one page on the stacksgov/pm issues now!

## M4 is in progress :microscope: 

After circulating the original survey, there was consistent feedback that we needed a less "western-centric" image for governance, and some additional ideas are under consideration. See [this issue comment](https://github.com/stacksgov/pm/issues/115#issuecomment-812045592) for more info.

Once we have a V2 set of basic designs, I will create a new survey and circulate it via Discord and the forum.

## M5 is ready to begin :factory_worker: 

This step takes a look at our shared account and sets up access procedures, i.e. how can I post videos to Stacksgov Youtube?

It's been great to watch these procedures evolve alongside the group, and once M5 is complete, M6 will take a look at everything as a whole and will represent the finished product.

**Comment by RaffiSapire** [comment]
Hey @whoabuddy , wanted to check in on this grant and estimated time to completion. Is there anything you need help with? Thank you!

**Comment by will-at-stacks** [comment]
Hello and thank you for participating in the Stacks Foundation Grants Program!

We are in the process of migrating from GitHub to the new [Grants Dashboard](https://grants.stacks.org/). In order to complete your grant, you will need to submit any remaining Progress Review and/or Final Review requests through the Dashboard in order to receive your remaining payments.

- If you have questions about the Grants Dashboard in general please email [ambassadors@stacks.org](mailto:ambassadors@stacks.org).
- If you have questions regarding payments and/or contracts please email [payments@stacks.org](mailto:payments@stacks.org).

Lastly, please note we are marking this grant 'closed' on GitHub for organizational purposes, but it is still 'open' on the Grants Dashboard.

Thanks and we hope to continue to support your efforts with additional grants!

Best,
Will
